onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -1883,5 +1885,40 @@ TEST(CApiTest, TestPerSessionCustomThreadPoolHooks) {
   ASSERT_TRUE(custom_join_hook_called == (thread_count - 1) << 1);
 }
 
+TEST(CApiTest, crop_and_resize) {
+  std::vector<float> input_value_0;
```

Comment:
Please add a comment capturing what exactly this is testing and why.

Reply:
Done.